airflow/providers/google/cloud/hooks/dataflow.py

```diff
@@ -252,7 +252,7 @@ def _get_current_jobs(self) -> list[dict]:
                 self._job_id = jobs[0]["id"]
             return jobs
         else:
-            raise Exception("Missing both dataflow job ID and name.")
+            raise AirflowException("Missing both dataflow job ID and name.")
```

Comment:
Could be a case of ValueError since these values need to be provided to the hook?

Reply:
Good point. I checked again and yes in makes sense for me to rather use `ValueError`. Changed.